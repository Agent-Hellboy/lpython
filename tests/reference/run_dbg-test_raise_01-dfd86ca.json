{
    "basename": "run_dbg-test_raise_01-dfd86ca",
    "cmd": "lpython {infile} -g --debug-with-line-column --no-color",
    "infile": "tests/runtime_errors/test_raise_01.py",
    "infile_hash": "359f595e813294fccdacb372c25a2aa6d3827066af7d91db62e32d83",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run_dbg-test_raise_01-dfd86ca.stderr",
    "stderr_hash": "dea131060a90f0b71a1794889ac07fd7658a27c4dd4da7df9e193dc2",
    "returncode": 1
}