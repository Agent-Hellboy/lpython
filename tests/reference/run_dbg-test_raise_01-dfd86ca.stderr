ld: warning: object file (test_raise_01.out.tmp.o) was built for newer macOS version (14.0) than being linked (13.3)
  File "tests/runtime_errors/test_raise_01.py", line 1
    def test():
  File "tests/runtime_errors/test_raise_01.py", line 4
    test()
  File "tests/runtime_errors/test_raise_01.py", line 2
    raise
ERROR STOP
